Development_Documents/Issue_Management_Process.md

```diff
@@ -0,0 +1,48 @@
+
+## 1. Creating Issues
+Whether submitting a new issue as a bug report or a feature request, the submission should contain the same general information and will follow the same process. The information to include in any new issue would be:
+- a title
+- a description
+- appropriate labels to indicate the BOINC components that the issue affects (these labels are prefixed by "C: ", e.g. "C: Manager" for the BOINC manager desktop software, or "C: Server - Validator" for the validator component of the BOINC server)
+
+The assignee should only be set when someone has taken ownership of the issue, as issues without an assignee will show up in a list of issues waiting for someone to begin work on them.
+
+### 1.1 Issue Review
+Issues should be reviewed prior to development beginning on them. Other members on the BOINC community should look for issues that have not been reviewed and determine, if they are able, whether an issue is suitable for development now, in the future, or not at all. The following labels determine the review status:
+- Verified: a good candidate for current development
+- Discussion Needed: bring up on the BOINC Contributors call for discussion
+- Parked: likely to be a candidate for development in the future, but not now
+
+Importantly, if an issue is not ever likely to be a candidate for development, it should be given a proper explanation and then be closed to prevent the backlog of issues from being overwhelmed in time.
+
+### 1.2 Milestones
+Milestones are used for organizing BOINC client and server releases. Only the release manager for a BOINC client or server release should add issues to a milestone (conversely, any pull request being merged into the master branch should be added to the milestone for the next release branch that has not been cut).
+
+### 1.3 Assignees
+The assignee of an issue indicates who has taken ownership of it and is responsible for seeing that it is carried to completion. Issues should not be arbitrarily assigned to users, as what is most important is willingness to work on the task.
+
+Any unassigned issues can be found using "no:assignee" in the issue search filter, which is valuable for developers looking for tasks that need to be done. For this reason, a user who is the assignee for an issue but is not willing to continue working on it should clear out the assignee field so that another developer can pick up the task (any final comments should also be made at this time to aid in the transition).
+
+## 2. Use Cases
+
+### 2.1 New Contributor
+A user looking to make first contributions to the BOINC community can use the "good first issue" label to find a list of issues that the community has deemed to be friendly ways to get involved and learn some of the ins and outs of BOINC development. 
```

Comment:
I started using the label _Newbie_ for this purpose. Should I change the label in git or should we change the label in the document?